### PR TITLE
Codecov + minor cleanup

### PR DESCRIPTION
**Pull request by dschwoerer**
I forgot I had the changes - should I split them as they are unrelated to the codecov change?

It addresses at least part of #26

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+comment: false
+
+ignore:
+  - "test"
+
+codecov:
+  branch: next
```
Comment (loeiten) [inline .codecov.yml:7]: Could we let it run on all branches?

File: boututils/fft_integrate.py [hunk 2]
```diff
@@ -8,42 +8,38 @@
...
+    n = np.size(y)
 
-        n = np.size(y)
+    f = old_div(np.fft.fft(y), n)
+    imag = np.complex(0.0, 1.0)
 
-        f = old_div(np.fft.fft(y), n)
-        imag = np.complex(0.0, 1.0)
+    result = np.arange(n) * f[0]
+    # loop = np.float(n) * f[0]  # return the loop integral
```
Comment (loeiten) [inline boututils/fft_integrate.py:17]: Maybe this could be removed as well?

**Review by loeiten — APPROVED** [review]
LGTM

**Review by loeiten — APPROVED** [review]
LGTM

**Comment by ZedThree** [comment]
The tests haven't run -- I think because Github disabled the actions because they had a `schedule` element and there was "no activity in the last 60 days" (from the Actions page).

I've re-enabled the actions, but I can't see a way to manually trigger them on this PR, so I'm going to push an empty commit. If that fails, I've turned off the branch protection for admins so I can force-merge this without the tests.

**Pull request merged**